test/JDBC/input/smalldatetimefromparts-vu-prepare.sql

```diff
@@ -0,0 +1,25 @@
+CREATE VIEW smalldatetimefromparts_vu_prepare_v1 as (SELECT SMALLDATETIMEFROMPARTS(2011, 8, 15, 14, 30));
```

Comment:
These are dependency tests. 
Please rename this file as `smalldatetimefromparts-dep-vu-prepare.sql` and it applies for other files as well.